careers/base/static/js/ga_event-tracking.js

```diff
@@ -93,6 +93,11 @@
     });
 
 
+    /* Careers > People
+    ***************************************************************************/
+    trackClick('.life-people-cta', ['How We Take Care of Our People', 'link click', 'Learn more']);
```

Comment:
I think it should be `How We Take Care of Our People Interactions` based on the other links being tracked.